proto/redpanda/core/admin/v2/shadow_link.proto

```diff
@@ -260,6 +260,16 @@ message TopicMetadataSyncOptions {
     // If this is true, then only the properties listed in
     // `synced_shadow_topic_properties` will be synced.
     bool exclude_default = 4;
+    // If set, the `source_topic_syncer` task will attempt to add the `_schemas`
+    // topic to the list of Shadow Topics as long as:
+    // 1. The `_schemas`` topic exists on the source clsuter
```

Comment:
few comments

* extra backtick "`schemas``"
* I feel the name of the config doesn't convey what it is doing, feels like it has to be an enum or something (enum values being the behavior the user wants).
* Since this is a user facing doc, remove references to implementation like `source_topic_syncer`? don't think the reader really cares. (or if it has to be a boolean, I think it should be something along the lines of `shadow_schemas_topic` or sth like that)